weed/filer/filer_delete_entry.go

```
@@ -63,7 +63,7 @@ func (f *Filer) DeleteEntryMetaAndData(ctx context.Context, p util.FullPath, isR
 
 	if isDeleteCollection {
 		collectionName := entry.Name()
-		f.DoDeleteCollection(collectionName)
+		f.DoDeleteCollection(ctx, collectionName)
```

golangci-lint (2.12.2)

[error] Error return value of `f.DoDeleteCollection` is not checked

(errcheck)